packages/migrate/src/commands/MigrateDiff.ts

```diff
@@ -0,0 +1,260 @@
+  ${chalk.dim('$')} prisma migrate diff \\
+    --preview-feature \\
+    --shadow-database-url "$SHADOW_DB" \\
+    --from-url "$PROD_DB" \\
+    --to-schema-datamodel=previous_datamodel.prisma \\
+    --script
+  
+  From a Prisma Migrate \`migrations\` directory to another database
+    e.g. generate a migration for a hotfix already applied on production
+  ${chalk.dim('$')} prisma migrate diff \\
+    --preview-feature \\
+    --shadow-database-url "$SHADOW_DB" \\
+    --from-migrations ./migrations \\
+    --to-url "$PROD_DB" \\
+    --script
+
+  Execute the --script output with \`prisma db execute\` using bash pipe \`|\`
+  ${chalk.dim('$')} prisma migrate diff \\
+  --preview-feature \\
+  --from-[...] \\
+  --to-[...]  \\
+  --script | prisma db execute --preview-feature --stdin --url="$DATABASE_URL"
+`)
+
+  public async parse(argv: string[]): Promise<string | Error> {
+    const args = arg(
+      argv,
+      {
+        '--help': Boolean,
+        '-h': '--help',
+        // From
+        '--from-empty': Boolean,
+        '--from-schema-datasource': String,
+        '--from-schema-datamodel': String,
+        '--from-url': String,
+        '--from-migrations': String,
+        // To
+        '--to-empty': Boolean,
+        '--to-schema-datasource': String,
+        '--to-schema-datamodel': String,
+        '--to-url': String,
+        '--to-migrations': String,
+        // Others
+        '--shadow-database-url': String,
+        '--script': Boolean,
+        '--preview-feature': Boolean,
+        '--telemetry-information': String,
+      },
+      false,
+    )
+
+    if (isError(args)) {
+      return this.help(args.message)
+    }
+
+    if (args['--help']) {
+      return this.help()
+    }
+
+    if (!args['--preview-feature']) {
+      throw new MigrateDiffNeedsPreviewFeatureFlagError()
+    }
+
+    const numberOfFromParamaterProvided =
+      Number(Boolean(args['--from-empty'])) +
+      Number(Boolean(args['--from-schema-datasource'])) +
+      Number(Boolean(args['--from-schema-datamodel'])) +
+      Number(Boolean(args['--from-url'])) +
+      Number(Boolean(args['--from-migrations']))
+
+    const numberOfToParamaterProvided =
+      Number(Boolean(args['--to-empty'])) +
+      Number(Boolean(args['--to-schema-datasource'])) +
+      Number(Boolean(args['--to-schema-datamodel'])) +
+      Number(Boolean(args['--to-url'])) +
+      Number(Boolean(args['--to-migrations']))
+
+    // One of --to or --from is required
+    if (numberOfFromParamaterProvided !== 1 || numberOfToParamaterProvided !== 1) {
+      const errorMessages: string[] = []
+      if (numberOfFromParamaterProvided !== 1) {
+        errorMessages.push(`${numberOfFromParamaterProvided} \`--from-...\` paramater(s) provided. 1 must be provided.`)
+      }
+      if (numberOfToParamaterProvided !== 1) {
+        errorMessages.push(`${numberOfToParamaterProvided} \`--to-...\` paramater(s) provided. 1 must be provided.`)
```

Comment:
```suggestion
        errorMessages.push(`${numberOfToParamaterProvided} \`--to-...\` parameter(s) provided. 1 must be provided.`)
```

Reply:
Variable can be renamed too.

Reply:
All done in https://github.com/prisma/prisma/pull/11440/commits/2780e8290d6f4102ae76a1926699858d2e1f2b68

Thanks!